README.md

```diff
@@ -192,6 +216,64 @@ Use the `Say` keyword to write the value of a variable to `STDOUT`.
 * `Say Tommy` - will output the value stored in `Tommy` to `STDOUT`
  
 Rockstar defines `Shout`, `Whisper` and `Scream` as aliases for `Say`
+
+The following examples all use c style syntax for explaining what things do.
+
+### Types Continued
+
+#### Operator Precedence
+
+The higher, the tighter the binding. This is the precedence we generally expect from our math.
+
+1. Function Call (greedy arguments)
+2. Logical NOT (right-associative)
+3. Multiplication and Division (left-associative)
+4. Addition and Subtraction (left-associative)
+5. Comparison operators (left-associative)
+6. `and`, `or`, and `nor` (left-associative)
+
+##### Examples
+
+- `A taking B times C plus not D times E and F` is equivalent to `((A(B) * C) + (!D * E)) && F`
+
+#### Binary Comparison
+
+Equality comparisons (`is`, `ain't`, `is not`) are allowed between types if they are the same type or they can be compared by the rules below. Objects are checked by reference equality, all other types are checked by value equality.
+
+Ordering comparisons (`is higher than`, `is lower than`, `is as high as`, and `is as low as`) are only allowed if the operands are both Numbers or both Strings or they are converted to such an arrangement according to the rules below. Numbers are compared as expected, Strings are compared lexicographically.
+
+- \<Mysterious\> \<op\> Mysterious =\> Equal.
```

Comment:
Probably doesn't need the `< >` if you're not gonna use them on the right side of the operation (same below with Non-Mysterious